### PR TITLE
Fixes #1459 - Include current version in Cocoapods Installation doc

### LINKED ISSUE #1459: Update Cocoapods Installation to include the current version
Without this, a previous version could by installed when the specs repository is out of date locally.

See [here](https://github.com/Moya/Moya/issues/1456#issuecomment-343477875).

Ref #1456

### PR DESCRIPTION
**Pull request by Snoops**
Resolves #1459

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -112,11 +112,11 @@ pod 'Moya'
 
 # or 
 
-pod 'Moya/RxSwift'
+pod 'Moya/RxSwift', '~> 10.0'
```
Comment (SD10) [inline Readme.md]: Any reason why we didn't do the core Moya pod as well?

Comment (Snoops) [reply Readme.md]: Hey, 

sorry, it is my first ever contribution and I may have misunderstood the issue. Is there another doc where Cocoapods Installation is mentioned and would need to be updated? From what I understood, it was simply to mention the version to use in the Cocoapods Installation section.

**Comment by ashfurrow** [comment]
Thanks a lot for contributing to Moya! I've invited you to join the Moya GitHub organization – no pressure to accept! If you'd like more information on what that means, check out our [contributor guidelines](https://github.com/Moya/contributors) and feel free to reach out with any questions.

**Comment by BasThomas** [comment]
Awesome, thanks @Snoops!

**Pull request merged**